UnityProject/Assets/Scripts/Items/Others/Candle.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+using Systems.Clothing;
+
+namespace Items.Others
+{
+	[RequireComponent(typeof(ItemLightControl))]
+	public class Candle : NetworkBehaviour, ICheckedInteractable<InventoryApply>, IServerDespawn
+	{
+		private ItemLightControl lightControl;
+
+		[SerializeField]
+		private SpriteHandler spriteHandler = default;
+
+		[SyncVar] public int LifeSpan = 120; //10 minutes
```

Comment:
Can you add it so if this is -1, then the candle is infinite? or maybe add another bool variable for that?